aws-api-payload.json

```diff
@@ -136,7 +136,7 @@
         "Configurations": [
             {
                 "Properties": {
-                    "yarn.nodemanager.remote-app-log-dir": "s3://$ADG_S3_LOG_BUCKET/yarn/",
+                    "yarn.nodemanager.remote-app-log-dir": "s3://$S3_LOG_BUCKET/yarn/",
```

Comment:
Should be `s3://$S3_LOG_BUCKET/emr/analytical-dataset-generation/yarn/`

Reply:
done